README.md

```diff
@@ -291,173 +321,204 @@ be used in global variable initializers.
 
 The `string.const` section indicates the literal as an `i32` index into
 a new custom section: a string table, encoded as a `vec(vec(u8))` of
-valid UTF-8 strings.  Because literal strings can contain codepoint 0,
+valid WTF-8 strings.  Because literal strings can contain codepoint 0,
```

Comment:
https://simonsapin.github.io/wtf-8/#intended-audience: "WTF-8 must not be used [...] for transmission over the Internet."
Bummer, but arguably finding a viable solution for the constraints we're facing in Wasm outweighs what some other spec did or didn't mean itself to be used for.

Reply:
Good catch!!  Filed #29, just so that we can formally decide one way ot the other.